spyder/utils/programs.py

```diff
@@ -817,84 +830,67 @@ def is_module_installed(module_name, version=None, installed_version=None,
     If version is not None, checking module version
     (module must have an attribute named '__version__')
 
-    version may starts with =, >=, > or < to specify the exact requirement ;
+    version may start with =, >=, > or < to specify the exact requirement ;
     multiple conditions may be separated by ';' (e.g. '>=0.13;<1.0')
 
     interpreter: check if a module is installed with a given version
     in a determined interpreter
     """
-    if interpreter:
-        if is_python_interpreter(interpreter):
-            checkver = inspect.getsource(check_version)
-            get_modver = inspect.getsource(get_module_version)
-            stable_ver = inspect.getsource(is_stable_version)
-            ismod_inst = inspect.getsource(is_module_installed)
+    if installed_version is not None:
+        '''
+        installed_version only passed in from dependencies.Dependency.check
+        which retrieved it from get_module_version, so we know the module is
+        already installed
+        '''
+        return True

```

Comment:
It seems now `installed_version` is unnecessary. Please remove it if that's the case.

Reply:
Okay, I'll look into it.

Reply:
Done